modules/rosa-autoscaling-nodes.adoc

```diff
@@ -47,3 +51,18 @@ Enable autoscaling on a machine pool with the ID `mp1` on a cluster named `myclu
 ----
 $ rosa edit machinepool --cluster=mycluster mp1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
+. To disable autoscaling on a machine pool, enter the following command:
```

Comment:
Disabling is a different use case from enabling, so I suggest moving this content to a different assembly. Even if you have to create a new one.

Reply:
Done.